### PR TITLE
Refresh 0.28

### PR DESCRIPTION
**Pull request by npanpaliya**
## Checklist before submitting

- [x] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/master/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/master/doc/)?
- [x] Did you write any [tests](https://github.com/open-ce/open-ce/blob/master/tests/) to validate this change?  

## Description

Fixes https://github.com/open-ce/open-ce/issues/183 and https://github.com/open-ce/DALI-feedstock/issues/10.

Some tests are still failing and I'm working on them. Also deactivate script is for some reason isn't working as expected. But this is same for TF too which I'll work on separately.

### CODE REVIEW

File: scripts/activate.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+COMPONENT="nvidia/dali"
+DALI_PKG_PATH=${CONDA_PREFIX}/lib/python${SYS_PYTHON_MAJOR}.${SYS_PYTHON_MINOR}/site-packages/$COMPONENT
+GREP=`type -P grep`
+
+# Will not set PATH - conda activate adds <environment>/bin to PATH
+
+# Will not set PYTHONPATH - Python path set by conda environment
+
+if ! echo $LD_LIBRARY_PATH | $GREP -q $DALI_PKG_PATH; then
+  LD_LIBRARY_PATH=$LD_LIBRARY_PATH${LD_LIBRARY_PATH:+:}$DALI_PKG_PATH
```
Comment (bnemanich) [inline scripts/activate.sh:24]: Why is this needed?

Comment (npanpaliya) [reply scripts/activate.sh:24]: There are a few shared libraries like `libdali_operators.so` of DALI which are needed for dali to function. So, to make those libs reachable for linker, this is needed.

Comment (bnemanich) [reply scripts/activate.sh:24]: OK, thanks!

**Review by bnemanich — APPROVED** [review]

**Review by cdeepali — APPROVED** [review]
